orcestra/postprocess/level1.py

```diff
@@ -111,7 +140,47 @@ def coarsen_radiometer(ds):
     return ds.coarsen(time=4, boundary="pad").mean().drop_duplicates("time")
 
 
-def filter_radar(ds, ds_bahamas):
+def resample_radiometer(ds, freq=pd.Timedelta("1s")):
+    """
+    Resample radiometer data to a given frequency using pandas dataframe resample method to speed up the process.
+
+    Parameters
+    ----------
+    ds : xarray.Dataset
+        level 0 radiometer data
+    freq : pandas.Timedelta
+        resampling frequency, default is 1 second
+
+    Returns
+    -------
+    xarray.Dataset
+        resampled radiometer data
+    """
+
+    TBs = [
+        ds.TBs.sel(frequency=f)
+        .to_dataframe()
```

Comment:
It still feels super weird that pandas resample should be faster... maybe we can have a look into that again